scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -188,16 +195,30 @@ task ExtractTask {
                 ~{true='--emit-pls' false='' emit_pls} \
                 ${FILTERING_ARGS}
 
-        du -b ~{output_file} | cut -f1 > vcf_bytes.txt
-        du -b ~{output_file}.tbi | cut -f1 > vcf_index_bytes.txt
+        INTERVAL_NUMBER=$(echo ~{output_file} | grep -oEi '[0-9]+\.vcf\.gz' | cut -d'.' -f1)
+
+        OUTPUT_FILE_BYTES=$(du -b ~{output_file} | cut -f1)
+        echo ${OUTPUT_FILE_BYTES} > vcf_bytes.txt
+
+        OUTPUT_FILE_INDEX_BYTES=$(du -b ~{output_file}.tbi | cut -f1)
+        echo ${OUTPUT_FILE_INDEX_BYTES} > vcf_index_bytes.txt
 
         # Drop trailing slash if one exists
         OUTPUT_GCS_DIR=$(echo ~{output_gcs_dir} | sed 's/\/$//')
 
         if [ -n "${OUTPUT_GCS_DIR}" ]; then
           gsutil cp ~{output_file} ${OUTPUT_GCS_DIR}/
           gsutil cp ~{output_file}.tbi ${OUTPUT_GCS_DIR}/
+          OUTPUT_FILE_DEST=${OUTPUT_GCS_DIR}/~{output_file}
+          OUTPUT_FILE_INDEX_DEST=${OUTPUT_GCS_DIR}/~{output_file}.tbi
+        else
+          OUTPUT_FILE_DEST=~{output_file}
+          OUTPUT_FILE_INDEX_DEST=~{output_file}.tbi
         fi
+
+        # Parent Task will collect manifest lines and create a joined file
+        # Currently, the schema is `[interval_number], [output_file_location], [output_file_size_bytes], [output_file_index_location], [output_file_size_bytes]`
+        echo ${INTERVAL_NUMBER},${OUTPUT_FILE_DEST},${OUTPUT_FILE_BYTES},${OUTPUT_FILE_INDEX_DEST},${OUTPUT_FILE_INDEX_BYTES} >> manifest.txt
```

Comment:
I'm not totally sold on this column's necessity. What are your thoughts? Is it needed in order to produce an ordered/merged manifest, i.e. for the `sort -n` later?

If we do need it, then I would use more generic terminology here, e.g. `shard_index` or `output_index`. I think the fact that this corresponds to a set of intervals currently is not relevant to or usable by an end consumer

Reply:
I'm OK with dropping it. I mostly added it for the sort functionality but also didn't think it would hurt to add.